src/test/java/de/tum/in/www1/artemis/exercise/quizexercise/QuizExerciseUtilService.java

```diff
@@ -851,4 +851,32 @@ public void joinQuizBatch(QuizExercise quizExercise, QuizBatch batch, String use
         user.setLogin(username);
         quizScheduleService.joinQuizBatch(quizExercise, batch, user);
     }
+
+    @NotNull
+    public QuizGroup createQuizGroup(String name) {
+        QuizGroup quizGroup = new QuizGroup();
+        quizGroup.setName(name);
+        return quizGroup;
+    }
+
+    @NotNull
+    public MultipleChoiceQuestion createMultipleChoiceQuestionWithTitleAndGroup(String title, QuizGroup quizGroup) {
+        return setQuizQuestionsTitleAndGroup(createMultipleChoiceQuestion(), title, quizGroup);
+    }
+
+    @NotNull
+    public DragAndDropQuestion createDragAndDropQuestionWithTitleAndGroup(String title, QuizGroup quizGroup) {
+        return setQuizQuestionsTitleAndGroup(createDragAndDropQuestion(), title, quizGroup);
+    }
+
+    @NotNull
+    public ShortAnswerQuestion createShortAnswerQuestionWithTitleAndGroup(String title, QuizGroup quizGroup) {
+        return setQuizQuestionsTitleAndGroup(createShortAnswerQuestion(), title, quizGroup);
+    }
+
+    private <Q extends QuizQuestion> Q setQuizQuestionsTitleAndGroup(Q quizQuestion, String title, QuizGroup quizGroup) {
```

Comment:
why are you returning the reference to the quiz. This is redundant.

Reply:
Thanks for noticing!